server/model/board_insights.go

```diff
@@ -4,12 +4,11 @@ import (
 	"encoding/json"
 	"io"
 
-	mmModel "github.com/mattermost/mattermost-server/v6/model"
+	mmModel "github.com/mattermost/mattermost/server/public/model"
 )
 
 // BoardInsightsList is a response type with pagination support.
 type BoardInsightsList struct {
```

Comment:
Boards insights seems that was also just used for Insights that has been deprecated. We should be able to delete this whole file as well.

Reply:
Done.